src/SprykerEco/Zed/Heidelpay/Communication/Plugin/Checkout/Oms/Condition/IsCaptureApprovedPlugin.php

```diff
@@ -33,43 +31,7 @@ class IsCaptureApprovedPlugin extends AbstractPlugin implements ConditionInterfa
      */
     public function check(SpySalesOrderItem $orderItem): bool
     {
```

Comment:
Create an `ItemTransfer` based on the `SpySalesOrderItem` and pass into the facade method.

Reply:
Seems like `idSalesOrder` is enough to check.

Reply:
We shouldn't use `SpySalesOrderItem` persistence types outside of the persistence layer. 
Please, fix it.